src/main/java/io/confluent/kafka/connect/datagen/DatagenTask.java

```diff
@@ -154,6 +157,12 @@ public void start(Map<String, String> props) {
       } catch (IllegalArgumentException e) {
         log.warn("Quickstart '{}' not found: ", quickstartName, e);
       }
+    } else if (schemaString != "") {
+      generator = new Generator.Builder()
```

Comment:
Do we need to copy this completely or can we just construct a single generator builder in this and the next branch, and then call `schemaString` or `schemaStream` on that builder as needed?